backend/apps/users/views.py

```
@@ -71,56 +70,39 @@ def google_login(request: HttpRequest, data: GoogleLoginRequest):
-        user.save()
+    user = get_or_create_google_user(User, email)
     backend = getattr(user, "backend", settings.AUTHENTICATION_BACKENDS[0])
     login(request, user, backend=backend)
     return AuthResponse(
         success=True,
         message="Logged in successfully.",
         user=UserResponse(id=user.id, email=user.email, is_active=user.is_active),
+    )
+    
+@router.post("/forgot-password", response=AuthResponse)
+def forgot_password(request: HttpRequest, data: LoginRequest):
```

Ruff (0.14.14)

[warning] Unused function argument: `request`

(ARG001)